src/components/ui/Table.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import Table from './Table';
+
+const columns = [
+  {
+    Header: ' ',
+    columns: [
+      {
+        Header: 'Test 1',
+        accessor: 'test1'
+      },
+      {
+        Header: 'Test 2',
+        accessor: 'test2'
+      }
+    ]
+  }
+];
+
+const data = [
+  {
+    test1: <span data-testid="test1">Data</span>,
+    test2: <span data-testid="test2">Data</span>
+  }
+];
+
+it('renders without crashing', () => {
+  const { getByTestId } = render(<Table columns={columns} data={data} />);
+  const TableComponent = getByTestId('Table-test');
+  expect(TableComponent).toBeInTheDocument();
+  expect(document.querySelector('[data-testid="test1"]')).toBeInTheDocument();
+  expect(document.querySelector('[data-testid="test2"]')).toBeInTheDocument();
+});
```

Comment:
One more test that comes to mind is - what happens when there are no data.

❓ And quick question - why not wrap the tests in `describe` block?

Reply:
```
 FAIL  src/components/common/Table/Table.test.js
  ● renders without crashing

    expect(received).toBeInTheDocument()

    received value must be an HTMLElement or an SVGElement.
    Received has value: null
```

The test fails if there is no data, of course 😎

I didn't add describe block because I used the default test that comes from CRA as a model, and it didn't have it. We could add describes in all the test if you want in a new card, but having tests in different files for each component I don't think is so necessary